File: Rust/fizzbuzz2.rs
const LIMIT: i32 = 100;

fn main() {
    for i in 1..=LIMIT {
        match i % 15 {
            0 => print!("FizzBuzz"),
            3 | 6 | 9 | 12 => print!("Fizz"),
            5 | 10 => print!("Buzz"),
            _ => print!("{}", i),
        }
        if i < LIMIT {
            print!(", ");
        }
    }
    println!();
}

File: OCaml/fizzbuzz2.ml
let limit = 100 in
for i = 1 to limit do
  (match i mod 15 with
  | 0 -> print_string "FizzBuzz"
  | 3 | 6 | 9 | 12 -> print_string "Fizz"
  | 5 | 10 -> print_string "Buzz"
  | _ -> print_int i);
  if i < limit then print_string ", " else print_newline ()
done
